src/main/java/lab5/test/players/LinusTest.java

```diff
@@ -0,0 +1,16 @@
+package players;
+
+import game.Board;
+import game.Position;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LinusTest {
+    @Test
+    void testLinusAlwaysPicksFirstAvailable() {
+        Board board = new Board(" XOXO  X ");
+        Linus linus = new Linus();
+        Position expectedMove = new Position(1, 1); // First available top-left position
+        assertEquals(expectedMove, linus.pickNextMove(board));
+    }
```

Comment:
This is a useful test, but here are more tests you should do.  See the sample solution.